Added Molpro 2019 support

batatasfritas opened the pull request.

batatasfritas committed:
Added Molpro 2019 support

rmsds commented on scitasexternal/packages/molpro/package.py:
```diff
@@ -5,24 +5,25 @@
 
     homepage = "http://www.molpro.net"
-    url      = "fake_url.tar.gz"
+    url      = 'file://%s/molpro-2019.2.tar.gz' % os.getcwd()
```
Will this work for `2015.1`?
If not we could use some variant of `self.version.up_to(2)`.

batatasfritas replied:
It will. I just tested it, to make sure. Since both versions have that exact numbering scheme, spack can find them, and we already put the files in the mirror anyway.

rmsds reviewed (changes requested):
Looks OK but I'm not sure about the `url =`

batatasfritas committed:
Minor "bugfix"

Modified the python dependency for backwards compatibility with the current installation of molpro.
